src/Costume.ts

```diff
@@ -1,5 +1,12 @@
 export default class Costume {
-  constructor(name, url, center = { x: 0, y: 0 }) {
+  public name: string;
+  public url: string;
+  public img: HTMLImageElement;
+  public isBitmap: boolean;
```

Comment:
Not an issue for this PR but let's revisit this `isBitmap` flag when dejankifying/reworking costume loading.